styles.css

```diff
@@ -1 +1,74 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+/*Grid mobile*/
+@media screen and (max-width: 800px) {
+    .container {
+        background-color: #fcf0de;
+        display: grid;
+        margin: 0;
+        grid-template-columns: 1fr;
+        grid-template-rows: repeat(7, minmax(50px, auto));
+        grid-gap: 16px;
+        grid-template-areas:
+    "a"
+    "b"
+    "c"
+    "d"
+    "e"
+    "."
+    "f";
+    }
+}
+
+.grid-item {
+    background-color: #ad4246;
+    display: flex;
+    color: white;
+    border: 1px solid #914042;
+    align-items: center;
+    justify-content: center;
+    padding: 15px;
+}
+
+.grid-a {
+    grid-area: a;
+}
+
+.grid-b {
+    grid-area: b;
+
+}
+
+.grid-c {
+    grid-area: c;
+}
+
+.grid-e {
+    grid-area: d;
+}
+
+.grid-d {
+    grid-area: e;
+}
+
+.grid-f {
+    grid-area: f;
+}
+
+/*Grid Desktop*/
+@media screen and (min-width: 800px) {
```

Comment:
Kijk nog even naar:
De media breakpoints maken gebruik van een minwidth ipv een max-width. Hierbij staat de code
voor de mobiele versie buiten de breakpoint, en
de code voor de web- en tablet-versie in de
breakpoint